README.md

```diff
@@ -1,2 +1,95 @@
-# navigation
-Repository for the discussion and research in to navigating from page to page whilst staying in immersive mode.
+# UA initiated immersive sessions
+
+## Motivation
+
+Current workflow to [request a WebXR session](https://www.w3.org/TR/webxr/#dom-xr-requestsession) requires user gesture activation. 2D Pages usually provide a button that users click or tap to trigger immersive mode.
+
```

Comment:
It should be noted that there has been plenty of discussion about how user-gesture is (alone) insufficient for entering VR.  We may require user-gesture, but my understanding is that we would still also require that the UA get user permission.  The user-gesture requirement is to avoid pages triggering XR mode automatically and annoying / surprising users.

Reply:
I've reworded the motivation. Tried to emphasize that we're not taking control from the user. User gesture requirement is specific to 2D pages and some workflows like navigation don't load a 2D page first. UA has the responsibility of informing and requesting appropriate permission to the user. Let me know what you think.